dwm1001/src/lib.rs

```diff
@@ -878,17 +818,16 @@ impl DW_IRQ {
     ///   DW1000.
     /// - This method disables interrupt handlers. No interrupt handler will be
     ///   called while this method is active.
-    pub fn wait_for_interrupts<T>(&mut self,
-        nvic:   &mut nrf52::NVIC,
+    pub fn wait_for_interrupts<T>(
+        &mut self,
+        nvic: &mut nrf52::NVIC,
         gpiote: &mut nrf52::GPIOTE,
-        timer:  &mut Timer<T>,
-    )
-        where T: TimerExt
+        timer: &mut Timer<T>,
+    ) where
+        T: TimerExt,
     {
         gpiote.config[0].write(|w| {
-            let w = w
-                .mode().event()
-                .polarity().lo_to_hi();
```

Comment:
Referring to my other comment about `cargo fmt`: Here's one example of code that got objectively worse (among many that I think got subjectively worse :-) ). With the manual formatting, the call that selected a field and the one that changes that fields value are grouped in a line. In the `cargo fmt` style, it's much harder to make out these grouping, making the code less readable.